source/test/Directory.Build.props

```diff
@@ -7,7 +7,9 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.0-preview-20200921-01" />
+		<PackageReference Include="FluentAssertions" Version="5.10.3" />
+		<PackageReference Update="Microsoft.NET.Test.Sdk" Version="16.9.0-preview-20201020-06" />
```

Comment:
I think there is now something askew - not sure if this was intended:
I'd have only `Include`s, and no `Update`s, and have all packages used in both test-projects in the `source/test/Directory.Build.props` - and have references which are exclusively used by a single project in the respective `*.Tests.csproj`.
Which would result in that we have every `PackageReference` at just one place, and update it in a single place for all descendants.

I will suggest `Directory.Packages.props` in our next session - perhaps this makes that more manageable.

Reply:
Don't mind me doing stupid things. Updated the packagereferences now correctly in the props. Never used that before and tbh also never will in my own private projects. 
I tend to see the csproj files as my single nuget source of truth since I don't trust the package manager ui. But maybe I just don't see the complete benefits of this approach for now so lets leave it as is :)